src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/EnvelopeParserTest.java

```diff
@@ -212,4 +215,18 @@ private JSONObject toJson(Document doc) throws Exception {
             .put("scanned_at", toIso8601(doc.scannedAt))
             .put("url", doc.url);
     }
+
+    private JSONArray toOcrJson(List<OcrDataField> ocrDataFields) throws JSONException {
+        JSONArray ocrJson = new JSONArray();
+        JSONObject ocrDataEntry;
+
+        for (OcrDataField ocrField : ocrDataFields) {
+            ocrDataEntry = new JSONObject();
+            ocrDataEntry.put("metadata_field_name", ocrField.name);
```

Comment:
Will this change the order ?

Reply:
No, this won't change the order

Reply:
We have a test in `OcrDataOrderTest` class, where we are testing the order of the fields.